Orange/widgets/visualize/owmosaic.py

```diff
@@ -837,7 +841,7 @@ def line(x1, y1, x2, y2):
                         if sum(counts) == 1:
```

Comment:
Marking the whole rectangle is ugly, unlikely and non-obvious (despite being here for at least three years). I suggest removing this `if` and its contents altogether.